pkg/apis/sealedsecrets/v1alpha1/sealedsecret_test.go

```diff
@@ -572,6 +572,33 @@ func TestRejectBothEncryptedDataAndDeprecatedV1Data(t *testing.T) {
 	}))
 }
 
+func TestInvalidBase64(t *testing.T) {
+	sealedSecret := &SealedSecret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "myname",
+			Namespace: "myns",
+		},
+		Spec: SealedSecretSpec{
+			EncryptedData: map[string]string{
+				"foo": "NOTVALIDBASE64",
+			},
+		},
+	}
+
+	scheme := runtime.NewScheme()
+	codecs := serializer.NewCodecFactory(scheme)
+	_, keys := generateTestKey(t, testRand(), 2048)
+
+	_, err := sealedSecret.Unseal(codecs, keys)
+	if err == nil {
+		t.Fatalf("Unseal did not return expected error: %v", err)
+	}
+
+	if !strings.Contains(err.Error(), "foo") {
+		t.Errorf("Unseal error did not contain field name %q: %v", "foo", err)
```

Comment:
For consistency with the other tests:

```suggestion
		t.Errorf("Expecting error: %v to contain field %q", err, "foo")
```

Reply:
changed